bridge-ffi/src/lib.rs

```diff
@@ -224,7 +262,7 @@ pub extern "C" fn tmprl_core_init(
 /// protobuf message. The callback is invoked on completion with a
 /// ShutdownResponse protobuf message.
 #[no_mangle]
-#[allow(unused_variables)]
+#[allow(unused_variables)] // We intentionally ignore the request
```

Comment:
Don't do this - just use a leading underscore for unused params

Reply:
That affects the header output, I'd like to keep that the way it is (these are Rust rules, they shouldn't affect C ergonomics)

Reply:
Riiight. K, if this annotation works being moved to apply to just the arg, then I'd do that, otherwise I'm fine with it.

Alternatively, maybe we just don't have the parameter at all? It's not like it's a huge lift to go refactor users of this package if we ever do add meaningful content to the proto. Not a huge deal though if we don't want to.

Reply:
I like the consistency. There's also a case of an empty return type too (shutdown). I kinda followed gRPC philosophy of "always have request and response even if empty" even though you're right, this is internal API and therefore easily refactored when we change.

Reply:
Rust does support this per param, so I set them there.